tasks/archive-install.yml

```diff
@@ -20,23 +20,19 @@
   # shell: "yes qy | hdiutil attach '{{ pkg_path | expanduser }}'"
   shell: "yes qy | hdiutil attach '{{ pkg_path }}' -mountpoint '{{ mount_path }}'"
   when: archive_extension == '.dmg' and mounted is defined and not mounted.stat.exists
-  changed_when: false
-
+  changed_when: False
 
 - block:
+  # we want the super-user to remvoe the existing app in case it was installed by another user
   - name: Remove existing app if present
     file:
       path: "{{ dest }}/{{app_basename}}"
```

Comment:
please ensure spaces inside curly brackets (lint)

Reply:
this is fixed, I also search for any other missing space after curly brackets